tests/harness/cases/maps.proto

```diff
@@ -17,7 +17,7 @@ message MapNoSparse {
     message Msg {}
 }
 
-message MapKeys   { map<sint32, string> val = 1 [(validate.rules).map.keys.sint32.lt = 0]; }
+message MapKeys   { map<sint64, string> val = 1 [(validate.rules).map.keys.sint64.lt = 0]; }
```

Comment:
@rmichela this triggers compilation error on `pgv-java-validation` without the fix - did you have another specific test case in mind?